connaisseur/trust_data.py

```diff
@@ -36,36 +38,23 @@ def __new__(cls, data: dict, role: str):
         try:
             return super(TrustData, cls).__new__(classes[role])
         except KeyError as err:
-            if re.match("^targets/[^/\\s]+$", role):
+            if re.match(r"^targets\/[^\/\s]+$", role):
```

Comment:
Is the `r` really what you want here? It massively changes the regex, which I wouldn't have expected from refactor

```python
>>> a = r"^targets\/[^\/\s]+$"
>>> a
'^targets\\/[^\\/\\s]+$'
>>> b = "^targets/[^/\\s]+$"
>>> b
'^targets/[^/\\s]+$'
```